### ISSUE TITLE
notcurses-info: first "capability" line duplicated

### ISSUE THREAD

**Issue by dnkl** [dankamongmen/notcurses#2650]
This appears to be a scrolling related issue, as it only happens when the output from `notcurses-info` requires scrolling the terminal content.

![duplicated-capability-line](https://user-images.githubusercontent.com/14031526/162035156-4df52ee7-2a64-4c1d-a169-8af7f98ba8b9.png)

See how the `af+ ab+...` line is duplicated?

Notcurses version: 3.0.8.
Reproduces in (at least): foot, xterm, kitty and alacritty

**Comment by dnkl** [comment]
Oh, and the emoji line is missing as well.

**Comment by dankamongmen** [comment]
yeah, scrolling output was broken in 3.0.5. it's my top priority to fix.
